docs/streams/stream-basics/subscriptions.md

```diff
@@ -1,14 +1,32 @@
 ---
-sidebar_position: 40
+sidebar_position: 200
 title: Subscriptions
 ---
 
 A subscription is a named configuration rule that determines how messages are delivered to consumers. There are three available subscription modes in GDN streams: [exclusive](#exclusive), [shared](#shared), and [failover](#failover). These modes are illustrated in the figure below.
 
 ![stream-subscription-modes](/img/stream-subscription-modes.png)
 
+To configure a subscription:
+
+1. Create a [producer](producers.md).
```

Comment:
@jamesm-macrometa should these `.md` extensions be removed?

Reply:
Thanks for the reminder, good catch.  It seems to build successfully either way, but I'll opt for removing the extensions since that's what we have written in our Style Guide.

Reply:
@dustinlarimer Yes, good catch.  Thanks!

Reply:
OK, I don't know why, but these particular links need the `.md` extensions to work.  This is frustratingly inconsistent, isn't it?  @diana-macrometa

Reply:
No clue. There must be a pattern... @dlozina-macrometa, any ideas?